libraries/Microsoft.Bot.Connector/Authentication/AdalAuthenticator.cs

```diff
@@ -15,6 +16,7 @@ namespace Microsoft.Bot.Connector.Authentication
     public class AdalAuthenticator : IAuthenticator
     {
         private const string MsalTemporarilyUnavailable = "temporarily_unavailable";
+        private const int HttpTooManyRequests = 429;
```

Comment:
>  private const int HttpTooManyRequests = 429; [](start = 6, length = 46)

don't need, you can use (int)HttpStatusCode.TooManyRequests

Reply:
HttpStatusCode.TooManyRequests became available in netstandard2.1 - this project is <TargetFramework>netstandard2.0</TargetFramework>:

https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode?view=netstandard-2.1#System_Net_HttpStatusCode_TooManyRequests

Reply:
Oh, snap! :)